src/ucp/api/ucp.h

```diff
@@ -2235,16 +2235,16 @@ ucs_status_t ucp_worker_signal(ucp_worker_h worker);
 
 /**
  * @ingroup UCP_WORKER
- * @brief Accept connections on a local address of the worker object.
+ * @brief Create a listener to accept connections on. Connection requests on
+ * the listener will arrive on a local address specified by the user.
  *
- * This routine binds the worker object to a @ref ucs_sock_addr_t sockaddr
+ * This routine binds the listener object to a @ref ucs_sock_addr_t sockaddr
```

Comment:
This routine creates a new listener object, that is bound to a specific local address.